Default (Linux).sublime-keymap

```diff
@@ -4,7 +4,7 @@
         "command": "typescript_paste_and_format",
         "context": [
             { "key": "setting.typescript_auto_format", "operator": "equal", "operand": true },
-            { "key": "selector", "operator": "equal", "operand": "source.ts" }
+            { "key": "selector", "operator": "equal", "operand": "source.ts, source.tsx" }
```

Comment:
Interesting. I didn't realize you could just use a comma separated list here. That's handy! :smile: